troposphere/guardduty.py

```diff
@@ -12,6 +12,7 @@ class Detector(AWSObject):
 
     props = {
         'Enable': (boolean, True),
+        'FindingPublishingFrequency': ([basestring], False),
```

Comment:
Looking at the docs, this likely should be `basestring` (string) instead of `[basestring]` (list of strings). Could you confirm?

Reply:
Yes, you are correct, it was a mistake on my part. I have corrected that now.